src/components/AppLoader.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
+import { SyncIndicator } from '@aragon/ui'
 import { useCourtConfig } from '../providers/CourtConfig'
 
 function AppLoader({ children }) {
   const courtConfig = useCourtConfig()
 
-  if (!courtConfig) return <p>Loading</p> // TODO: Add better loader
+  if (!courtConfig) return <SyncIndicator visible /> // TODO: Add better loader
```

Comment:
This was meant as a placeholder since it was intended to be like the app loader to ensure that the child components that use the court config can make sure that it is not `undefined` since is data that is necessary across all the app. 

Maybe what we could do is show the <SyncIndicator/> on every main component (Dashboard, Tasks, Disputes) when the necessary data has loaded ?